frontend/src/metabase/parameters/components/ValuesSourceSettings/ValuesSourceSettings.tsx

```diff
@@ -23,79 +21,92 @@ export interface ValuesSourceSettingsProps {
   ) => void;
 }
 
-const ValuesSourceSettings = ({
+export function ValuesSourceSettings({
   parameter,
   onChangeQueryType,
   onChangeSourceSettings,
-}: ValuesSourceSettingsProps): JSX.Element => {
+}: ValuesSourceSettingsProps) {
   const queryType = getQueryType(parameter);
   const [isModalOpened, setIsModalOpened] = useState(false);
 
-  const radioOptions = useMemo(() => {
-    return getRadioOptions({
-      queryType: queryType,
-      onEditClick: () => setIsModalOpened(true),
-      // linked filters only work with connected field sources (metabase#33892)
-      isEditDisabled: hasLinkedFilters(parameter),
-    });
-  }, [queryType, parameter]);
-
-  const handleModalClose = useCallback(() => {
-    setIsModalOpened(false);
-  }, []);
+  // linked filters only work with connected field sources (metabase#33892)
+  const disableEdit = hasLinkedFilters(parameter);
+  const openModal = () => setIsModalOpened(true);
+  const closeModal = () => setIsModalOpened(false);
 
   return (
     <>
-      <Radio
-        value={queryType}
-        options={radioOptions}
-        vertical
-        onChange={onChangeQueryType}
-      />
+      <Radio.Group value={queryType} onChange={onChangeQueryType}>
+        <Stack spacing="xs">
+          <RadioLine
```

Comment:
What does the `Line` suffix mean? Did you want to communicate it's a [block](https://developer.mozilla.org/en-US/docs/Glossary/Block-level_content)?

How about communicating the purpose instead of including presentation details in the name?

```suggestion
          <QueryTypeRadio
```

Reply:
I agree it was a bit confusing. It meant "form line", but it's not the best name. Will rename.